lib/workers/repository/update/pr/code-owners.spec.ts

```diff
@@ -22,13 +23,183 @@ describe('workers/repository/update/pr/code-owners', () => {
       expect(codeOwners).toEqual(['@jimmy']);
     });
 
+    it('respects orphan files', async () => {
+      fs.readLocalFile.mockResolvedValueOnce(
+        ['* @jimmy', 'yarn.lock'].join('\n')
```

Comment:
please use codeBlock format instead

Reply:
Solved in c2888e3faa6c3ff7a3c74f5ea227b8ae4e19a92b